scripts/msmtpq/msmtpq

```diff
@@ -55,22 +82,44 @@ shopt -s inherit_errexit
 IFS=$' \n\t'
 PS4='+\t '
 
-if [[ -z "$NOTIFY_SEND" ]] && [[ ! -t 0 ]] && [[ -n "$DISPLAY" ]] && command -v notify-send >/dev/null 2>&1; then
+if [ -n "$NOTIFY_SEND_VERBOSE" ]; then
+  NOTIFY_SEND=1
+fi
+
+if [ -z "$NOTIFY_SEND" ] && [ ! -t 0 ] && [ -n "$DISPLAY" ] && command -v notify-send >/dev/null 2>&1; then
   NOTIFY_SEND=1
 fi
 
+if [ "$NOTIFY_SEND" = 1 ]; then
+  if [ -z "$DISPLAY" ]; then
+    err "NOTIFY_SEND=1 set but no display available!"
```

Comment:
I don't think this is the right way to handle this. If anything, it should just ignore it instead of erroring out.

But even better, just remove this special case. Just because `$DISPLAY` is unset, doesn't mean that notifications won't be working. For example on Wayland `$DISPLAY` is not set in general.